content/en/intelligent_test_runner/setup/python.md

```diff
@@ -39,6 +39,17 @@ Prior to setting up Intelligent Test Runner, set up [Test Visibility for Python]
 
 {{% ci-itr-activation-instructions %}}
 
+### Required dependencies
+
+The Intelligent Test Runner requires the [`coverage` package][2].
+
+Install it in your environment by specifying it in your `requirements.txt` (for example), or using `pip`:
```

Comment:
```suggestion
Install the package in your environment by specifying it in your `requirements.txt` for example, or using `pip`:
```

Reply:
That's what I originally had before putting up the PR and then thought "I'm saying package twice in two sentences, it feels redundant." 😅